ras-onboarding/ras_onboarding/namespace/queries.py

```
@@ -11,8 +23,61 @@
                                updated_by = 'system' \
```

i believe the tailing slash can be removed... did this query run properly? i believe it should throw a syntax error right? 🤔